pkg/collector/py/check_test.go

```diff
@@ -119,8 +120,15 @@ func TestInitNewSignatureCheck(t *testing.T) {
 }
 
 func TestInitException(t *testing.T) {
+	var expectedErrString string
 	_, err := getCheckInstance("init_exception", "TestCheck")
-	assert.EqualError(t, err, "could not invoke python check constructor: ['Traceback (most recent call last):\\n', '  File \"tests/init_exception.py\", line 11, in __init__\\n    raise RuntimeError(\"unexpected error\")\\n', 'RuntimeError: unexpected error\\n']")
+
+	if runtime.GOOS == "windows" {
+		expectedErrString = "could not invoke python check constructor: ['Traceback (most recent call last):\\n', '  File \"tests\\\\init_exception.py\", line 11, in __init__\\n    raise RuntimeError(\"unexpected error\")\\n', 'RuntimeError: unexpected error\\n']"
+	} else {
+		expectedErrString = "could not invoke python check constructor: ['Traceback (most recent call last):\\n', '  File \"tests/init_exception.py\", line 11, in __init__\\n    raise RuntimeError(\"unexpected error\")\\n', 'RuntimeError: unexpected error\\n']"
+	}
+	assert.EqualError(t, err, expectedErrString)
```

Comment:
instead of this conditional block can we do simply:
```
assert.Contains(t, err.String(), "RuntimeError: unexpected error")
```
I know it's not exactly the same but I think the tradeoff for a more readable code is worth it